p2p/gater.go

```diff
@@ -26,19 +26,15 @@ import (
 var _ connmgr.ConnectionGater = ConnGater{}
 
 // NewConnGater return a new connection gater that limits access to the cluster peers and relays.
-func NewConnGater(peers []peer.ID, relays []Peer) (ConnGater, error) {
+func NewConnGater(peers []peer.ID, relays []*MutablePeer) (ConnGater, error) {
 	peerMap := make(map[peer.ID]bool)
 	for _, peerID := range peers {
 		peerMap[peerID] = true
 	}
 
-	// Allow connections to/from relays.
-	for _, relay := range relays {
-		peerMap[relay.ID] = true
-	}
```

Comment:
don't think this is required since it auto adds relay addresses from circuit multiaddrs